src/Introduction.mdx

```diff
@@ -0,0 +1,46 @@
+# Front End Interview — Design System
+
+## Hi and welcome to our code challenge. 👋
+
+Make sure to carefully read the following brief, containing all the details you need.
+
+## The Goal
+
+For this take-home test, the goal is to develop a Tabs component that will be part of our Design System project.
+The component should be accessible and reusable, following best practices in front-end development.
+
+## Acceptance Criteria
+
+Share with us the 🔗 link to your repository (don’t fork please) that contains everything you need to show us about the component and its features.
+More exactly, we write some user stories:
```

Comment:
Not sure it is hard to read and understand: Maybe I would do: 
## The Goal 
For this take-home test, the goal is to develop a Tabs component that will be part of our Design System project.
The component should be accessible and reusable, following best practices in front-end development.
At the end of the test share with us the 🔗 link to your repository that contains everything you need to show us about the component and its features (don’t fork please).

## Acceptance Criteria
You need to develop the tab component following the acceptance criteria  which we write in the form of a user story